pulsar-client-cpp/python/pulsar/functions/serde.py

```diff
@@ -68,9 +68,9 @@ def deserialize(self, input_bytes):
       return pickle.loads(input_bytes)
 
 class IdentitySerDe(SerDe):
-  """Pickle based serializer"""
+  """utf-8 based identity serializer"""
   def serialize(self, input):
-    return input
+    return bytes(str(input).encode('utf-8'))
```

Comment:
how can we assume that the string representation of the object is meaningful?  The identity serde is also used as the default serde in python which is going to cause errors if users inputs/outputs anything other than Strings

Reply:
by default the string representation of an object is:

>>> str(foo)
'<__main__.Foo instance at 0x10631d320>'